phpBB/styles/prosilver/composer.json

```
@@ -0,0 +1,25 @@
+{
+	"name": "phpbb/phpbb-style-prosilver",
+	"description": "phpBB Forum Software default style",
+	"type": "phpbb-style",
+	"version": "4.0.0-a1-dev",
+	"homepage": "https://www.phpbb.com",
+	"license": "GPL-2.0",
+	"authors": [
+		{
+			"name": "phpBB Limited",
+			"email": "operations@phpbb.com",
+			"homepage": "https://www.phpbb.com/go/authors"
+		}
+	],
+	"support": {
+		"issues": "https://tracker.phpbb.com",
+		"forum": "https://www.phpbb.com/community/",
+		"wiki": "https://wiki.phpbb.com",
+		"irc": "irc://irc.freenode.org/phpbb"
+	},
+	"extra": {
+		"display-name": "prosilver",
+		"phpbb-version": "4.0.0-a1-dev"
+	}
```

May I know how will Style Parent will be Handled ???
Unless you are missing this part :
```suggestion
		"style-parent": "prosilver"
	}
```

Best regards 👍

---

That's actually a good point, I think it might make sense to explicitly state the parent here. Of course this would be empty in the case of prosilver as prosilver does not have a parent.

@mrgoldy thoughts?

---

> ... Of course this would be empty in the case of prosilver as prosilver does not have a parent. ...

Yea!
I went as-per `style.cfg` which has parent as it's self.
Either should do as long as it is mentioned.

Best regards 👍